### DISCUSSION TITLE
can users press `j` or `k` to move down/up in the menu?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [inducer/pudb#585] [Q&A]
![screen-2023-01-20-17-49-16](https://user-images.githubusercontent.com/32936898/213666640-e9b94f97-d2e9-4880-9d3a-eae4e7f3a825.jpg)

I notice

```python
[
    ("Restart", "restart"),
    ("Examine", "examine"),
    ("Quit", "quit"),
    ],
focus_buttons=True,
bind_enter_esc=False,
title="Finished",
extra_bindings=[
    ("q", "quit"),
    ("esc", "examine"),
    ])
```

So press `q` can directly `quit`.

However, can users press `j` or `k` to move down/up in this situation?

**Comment by inducer** [comment]
You can't at this moment, and I don't think it would be wise to address that inconsistency.

**Reply by Freed-Wu** [reply]
Oh, I just think press `<up>`/`<down>` is not convenient with `j`/`k` or `<C-p>`/`<C-n>`. Why it is unwise?

**Reply by inducer** [reply]
- It wouldn't work on text entry fields.
- Implementing it consistently would require reaching into Urwid at a fairly low level.
